api/test/integration/test_cluster_endpoints.tavern.yaml

```diff
@@ -1578,7 +1578,7 @@ stages:
           total_failed_items: 0
 
 ---
-test_name: GET /cluster/{node_id}/logs
+test_name: GET /cluster/{node_id}/daemons/stats
```

Comment:
These differences are because I moved the logs tests to the end of the file. Setting wazuh-modulesd to debug mode caused other logs to not appear until later in the execution and that caused some cases to fail. I think these tests should be run at the end so we get the maximum amount of logs possible.